tests/unit/amo/components/TestAddonHead.js

```diff
@@ -84,113 +85,50 @@ describe(__filename, () => {
     );
   });
 
-  it('renders a description meta tag containing the add-on summary', () => {
+  it('renders a HeadMetaTags component', () => {
     const addon = createInternalAddon(fakeAddon);
-    const root = render({ addon });
-
-    expect(root.find('meta[name="description"]')).toHaveLength(1);
-    expect(root.find('meta[name="description"]')).toHaveProp(
-      'content',
-      `Download ${addon.name} for Firefox. ${addon.summary}`,
-    );
-  });
-
-  it('renders Open Graph meta tags', () => {
     const lang = 'fr';
-    const addon = createInternalAddon(fakeAddon);
     const { store } = dispatchClientMetadata({ lang });
-    const root = render({ addon, store });
 
-    [
-      ['og:type', 'website'],
-      ['og:url', addon.url],
-      ['og:locale', lang],
-      ['og:image', addon.previews[0].image_url],
-    ].forEach(([property, expectedValue]) => {
-      expect(root.find(`meta[property="${property}"]`)).toHaveProp(
-        'content',
-        expectedValue,
-      );
-    });
+    const root = render({ addon, store });
 
-    expect(root.find(`meta[property="og:title"]`).prop('content')).toContain(
-      addon.name,
+    expect(root.find(HeadMetaTags)).toHaveLength(1);
+    expect(root.find(HeadMetaTags)).toHaveProp('date', addon.created);
+    expect(root.find(HeadMetaTags)).toHaveProp(
+      'description',
+      `Download ${addon.name} for Firefox. ${addon.summary}`,
+    );
+    expect(root.find(HeadMetaTags)).toHaveProp(
+      'image',
+      addon.previews[0].image_url,
+    );
+    expect(root.find(HeadMetaTags)).toHaveProp(
+      'lastModified',
+      addon.last_updated,
+    );
+    expect(root.find(HeadMetaTags)).toHaveProp(
+      'title',
+      `${addon.name} – Get this Extension for 🦊 Firefox Android (${lang})`,
     );
-    expect(
-      root.find(`meta[property="og:description"]`).prop('content'),
-    ).toContain(addon.summary);
-  });
-
-  it('does not render a "og:image" meta tag if add-on has no previews', () => {
```

Comment:
I'm not sure that this case is explicitly covered anymore by the new tests.

Reply:
I don't think this explicit case is covered by the `HeadMetaTags` suite, because it just checks for `image` being `null`, but after digging deeper I agree it's no longer needed because it's covered by https://github.com/mozilla/addons-frontend/blob/2018.11.08/tests/unit/core/test_imageUtils.js#L82-L89.